### PR TITLE
ci(travis): Enforce that code is formatted and that credo shows no warnings with elixir 1.9.1

### PR DESCRIPTION
**Pull request by ekstrom**
Added elixir 1.9.1 the list of versions to test.
Added a style guide section in the readme.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,12 @@ elixir:
   - 1.6.6
   - 1.7.4
   - 1.8.2
+  - 1.9.1
+
+jobs:
+  include:
+    - stage: check style guide
```
Comment (uesteibar) [inline .travis.yml]: Can we call it `lint` instead?

Comment (ekstrom) [reply .travis.yml]: That sounds good aswell. I'll change it. :smiley: Should the readme also be changed back into something like `To run the linters`? Similar how it previously looked.

Comment (uesteibar) [reply .travis.yml]: I think the readme is fine just like that! Thanks for taking care :)

Comment (ekstrom) [reply .travis.yml]: Ok, I changed the name of the stage to lint. Thanks for the quick reply :smiley:

**Comment by uesteibar** [comment]
Thanks for this, great addition!

**Pull request merged**